tests/dependencies/updater_test.php

```diff
@@ -608,37 +608,17 @@ public function testUpdateWithBrokenJson()
         );
 
         ob_start();
-        $result = $dependencies->update();
+        $result = $dependencies->updateAll();
         $output = ob_get_clean();
 
         $this->assertTrue($result->isErr());
         $this->assertEquals('missing sources', $result->getErr());
         $this->assertEquals("[Updating themes/my-theme]\n", $output);
     }
 
-    public function testUpdateDependencyWithNoLock()
+    public function testUpdateSingleWithNoLock()
     {
         $dir = $this->getDir();
-        $whippetJson = $this->getWhippetJson([
-            'src' => [
-                'plugins' => 'git@git.dxw.net:wordpress-plugins/',
-            ],
-            'themes' => [
-                [
-                    'name' => 'my-theme',
-                    'ref' => 'v1.4',
-                ],
-            ],
-            'plugins' => [
-                [
-                    'name' => 'my-plugin',
-                    'ref' => 'v1.6',
-                ],
-            ],
-        ]);
-        $this->addFactoryCallStatic('\\Dxw\\Whippet\\Files\\WhippetJson', 'fromFile', $dir.'/whippet.json', \Result\Result::ok($whippetJson));
-
-        file_put_contents($dir.'/whippet.json', 'foobar');
```

Comment:
Can you unpack what's happening here? Looks like this test used to set up a manifest file in a directory and use that in the test - now it doesn't use a manifest file at all?

I feel like this deserves a comment in the commit message.